src/Tinkoff.ISA.sln.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
```

Comment:
do we need this file? maybe we can add in gitignore?

Reply:
My aim was to share the word "Tinkoff", add it to the repo dictionary. Otherwise, it's highlighted as "typo in identifier".